site/src/examples/stepped-tracker/StepProgression.tsx

```diff
@@ -5,27 +5,27 @@ import { RefreshIcon } from "@salt-ds/icons";
 
 type Step = {
   label: string;
-  state: "default" | "completed";
+  state: "pending" | "completed";
 };
 
 type Steps = Step[];
 
 const sampleSteps: Steps = [
   {
     label: "Step One",
-    state: "default",
+    state: "pending",
```

Comment:
Should this be updated to follow the new naming?

Reply:
Part of me thinks that we should keep it as-is because app an app developer can define their state any way they like as long as it maps to the correct props 🤷 

Probably less confusing for us to use `stage` in our examples, though, so I'll update it